scripts/travisci/check_flake8.sh

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env bash
+# ABSOLUTE VERIFICATION:
+# The following errors are analyzed in all python files across the repository.
+errors_absolute=(
```

Comment:
can you include a list of errors which are ignored? that would make the file self-documenting.